convex/testing/clearAll.ts

```diff
@@ -0,0 +1,11 @@
+import { testingMutation } from "./wrappers";
+import schema from "../schema";
+
+export default testingMutation(async ({ db }) => {
+    for (const table of Object.keys(schema.tables)) {
+      const docs = await db.query(table as any).collect();
```

Comment:
would be great to get typechecking working here (not important though)

Reply:
It's the `Object.keys` that gives us `string` but maybe there's a different way to iterate through